src/exports/la-utils/la-transactions/handlers/sponsoredGas/sponsoredGasContractCall.ts

```diff
@@ -152,37 +154,44 @@ export const sponsoredGasContractCall = async ({
 
   // send the user operation with EIP-7702 delegation in a runOnce
   // so that we don't submit it more than once
-  const uoHash = await Lit.Actions.runOnce(
+  const sendWithAlchemyResult = await Lit.Actions.runOnce(
     {
       waitForResponse: true,
       name: "sendWithAlchemy",
     },
     async () => {
       try {
         // Send the user operation with EIP-7702 delegation
-        const userOpResult = await smartAccountClient.sendRawUserOperation(
+        const userOpHash = await smartAccountClient.sendRawUserOperation(
           signedUserOperation,
           entryPoint.address
         );
 
         console.log(
           `[@lit-protocol/vincent-tool-morpho/executeOperationWithGasSponsorship] User operation sent`,
-          { userOpHash: userOpResult }
+          { userOpHash }
         );
 
-        return userOpResult;
+        const bundleHash = eip7702WaitForBundleTx ? await smartAccountClient.waitForUserOperationTransaction({
```

Comment:
This can throw timeout, in such case the userOp might still pass so we want to return it anyway and let the user decide what they will do (probably just wait until they are sure the tx has passed or check the side effects)
If we throw when having a userOp then we will be hiding from the user that their tx might succeed later